src/Lockedfield.php

```diff
@@ -221,6 +237,101 @@ public function getForbiddenStandardMassiveAction()
 
     public static function canPurge()
     {
+        return Session::haveRight(static::$rightname, UPDATE);
+    }
+
+    public function prepareInputForAdd($input)
+    {
+        return $this->prepareInput($input);
+    }
+
+    public function prepareInputForUpdate($input)
+    {
+        return $this->prepareInput($input);
+    }
+
+    protected function prepareInput($input)
+    {
+        if (isset($input['item'])) {
+            list($itemtype, $field) = explode(' - ', $input['item']);
+            $input['itemtype'] = $itemtype;
+            $input['items_id'] = 0;
+            $input['field'] = $field;
+            $input['is_global'] = 1;
+        }
+
+        return $input;
+    }
+
+    public function showForm($ID, array $options = [])
+    {
+        $this->initForm($ID, $options);
+        TemplateRenderer::getInstance()->display('pages/admin/inventory/lockedfield.html.twig', [
```

Comment:
```suggestion
         unset($this->fields['is_global']);
         TemplateRenderer::getInstance()->display('pages/admin/inventory/lockedfield.html.twig', [
```

This to avoid the `is_global` field display (is_global is normally used by Monitor/Peripheral/Printer to have a generic item), as it make no sense in this context